lib/wallaby/element.ex

```diff
@@ -168,8 +171,9 @@ defmodule Wallaby.Element do
     case driver.send_keys(element, keys) do
       {:ok, _} ->
         element
-      {:error, :stale_reference_error} ->
+      {:error, :stale_reference} ->
         raise Wallaby.StaleReferenceException
+      e -> e
```

Comment:
I know I'm pedantic, but I'd like thos to say... element or error, whichever it is :D